handbook/documentation.rst

```diff
@@ -0,0 +1,33 @@
+####
+Developer Documentation
+####
+
+Overview
+********
+
+* Keep developer documentation as close to the thing you are describing as possible to make it easier to find and maintain.
+
+Details
+****
+
+* READMEs, HOW-TOs, API Docs and ADRs should live with the code.
+
+  * API docs should be in swagger format.
+
+* **Everything else should be RSTs**.  This_ is a good resources for learning RST.
+
+* Process docs such as scrum notes, retros, personal notes should not live in the repo and should generally be in confluence.
```

Comment:
Process docs should live where the team is comfortable with them. We should avoid applying restrictions on how teams function.

Reply:
Document editors in Confluence are not great and are challenging for asynchronous and team-based editing activities.

Reply:
@staubina that's true which is why this is a `should` instead of a `must`.  We want to make sure there is a clear default if you aren't sure of what you should be doing.